generate/generate.go

```diff
@@ -782,6 +782,25 @@ func (s *service) generateHelperFuncs(a *API) {
 			pn("		return nil, -1, err")
 			pn("	}")
 			pn("")
+			pn("    if l.Count == 0 {")
+			pn("		// look	inside projects ")
+			pn("		p.p[\"projectid\"] = \"-1\"")
+			pn("		l2, err := s.ListVirtualMachines(p)")
```

Comment:
I think it would be cleaner to just assign the result to the existing variables like this:

```go
pn("		l, err = s.ListVirtualMachines(p)")
```

This shouldn't be a problem as the count of `l` was zero, so doesn't contain any results.